src/shell/sidebar/Sidebar.js

```diff
@@ -115,15 +109,13 @@ class Sidebar extends Component {
     // We use onMouseOver instead of onMouseEnter because the user might click the avatar/links
     // while still in the sidebar. This way, the sidebar will open again as soon as another element
     // is hovered
-    handleMouseOver = (event) => {
+    handleMouseOver = () => {
         // This is needed because of event bubbling. Events inside a portal will be propagated
         // to its ancestors in the React tree (event though those elements are not ancestors in the DOM tree).
         // You can read more about it here: https://reactjs.org/docs/portals.html#event-bubbling-through-portals
-        const fromModal = !!event.target.closest('[role="dialog"]');
+        const isAddIdentityModalMounted = findDOMNode(this.addIdentityModalRef.current);
```

Comment:
I add to change the way we were checking this because it was being fired inside the lock screen.